data/json/items/tools.json

```diff
@@ -5610,7 +5610,31 @@
     "category" : "weapons",
     "use_action": { "type": "knife" }
   },
-
+  {
+    "id": "glaive",
```

Comment:
It's a dedicated weapon, not a tool.
Many weapons have use_actions, usually "knife".
Could lose that knife action by the way. It would be very unwieldy for that.